modules/acs-requirements.adoc

```diff
@@ -42,7 +42,10 @@ Starting with {product-title-short} 4.3, both Central and secured cluster servic
 See the default memory and CPU requirements for each component and ensure that the node size can support them.
 ====
 
-Persistent storage by using persistent volume claim (PVC). A PVC is required on the the cluster where Central is installed. It is strongly recommended on the secured clusters where Scanner V4 is enabled.
+Persistent storage by using persistent volume claim (PVC). A PVC is required on the the cluster where Central is installed. It is strongly recommended on the secured clusters where Scanner V4 (Technology Preview) is enabled.
```

Comment:
Same comment with thinking that "(Technology Preview)" isn't necessary since the TP notice directly follows. I won't point out any more instances of this.